_inc/client/components/settings-card/index.jsx

```diff
@@ -26,35 +27,42 @@ import Banner from 'components/banner';
 import Button from 'components/button';
 
 export const SettingsCard = props => {
-	let module = props.module
+	const module = props.module
 			? props.getModule( props.module )
-			: false,
-		header = props.header
-			? props.header
-			: '',
-		isSaving = props.isSavingAnyOption(),
+			: false;
+
+	// Non admin users only get Publicize, After the Deadline, and Post by Email settings. The UI doesn't have settings for Publicize.
+	// composing is not a module slug but it's used so the Composing card is rendered to show AtD.
+	if ( ! props.userCanManageModules && ! includes( [ 'composing', 'post-by-email' ], props.module ) ) {
+		return <span />;
```

Comment:
That would be possible if we were using React 15 which allows this in stateless components. It's not possible in .14 that we're currently using.